communication/slack_channels.md

```diff
@@ -0,0 +1,62 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels).
+
+### General channels
+
+* `#general`
+* `#offtopic`
+* `#officechat`
+  * Physical office related discussions
+* `#offtopic-`
+  * Search in slack for `#offtopic-` and you'll find different offtopic channels like `#offtopic-music`
+* `#osd`
+  * `#osd` stands for Open Source Days, people share what they'll be working on during their OSD and share the results/demos
+* `#twitter`
+  * Source{d} mentions and tweets
+
+### Team channels
+
+* `#eng-team`
+* `#product-team`
+* `#dev-infrastructure`
+* `#dev-retrieval`
+* `#dev-language-analysis`
+* `#dev-machine-learning`
+* `#dev-applications`
+
+### Daily standup channels
+
+This is where the daily standups happen.
+
+* `#su-general` 
+  * For those of us whose team is too small to warrant its own channel, e.g. CEO
+* `#su-product-team`
+* `#su-infrastructure`
+* `#su-data-retrieval`
+* `#su-language-analysis`
+* `#su-machine-learning`
+* `#su-applications`
+
+### Public channels that are shared with the community slack
```

Comment:
Worth thinking/describing about the purpose and content of these channels, since they're listed here and a good heads up of what is to be expected when joining the community chat.

Reply:
Also meaningful to link to the community invitation URL and encourage readers to join it: https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM

Reply:
+1